docker-compose-net/docker-compose-net.md

```diff
@@ -0,0 +1,234 @@
+---
+layout: guides
+title: How to Use Docker Compose with Weave Net
+---
+
+Docker Compose is used to configure multiple container apps. A Docker Compose file configures your application's services and with a single command creates and starts services. Depending on how you are using Weave Net, whether through the Docker API Proxy or by using the Weave Docker Plugin, the syntax of the Docker Compose file differs. This guide provides an overview of the different ways in which networking can be configured and also provides examples of the Docker Compose files for Weave Net. 
+
+The following topics are discussed: 
+
+  * [What You Will Use](#what-use)
+  * [Before You Begin](#before)
+  * [Attaching Containers to a Weave Network](#attaching-containers)
+  * [Docker Compose and the Weave Net Docker API Proxy](#setup-proxy)
+     * [Docker Compose File Syntax for the Weave Docker API Proxy](#compose-proxy)
+  * [Docker Compose and the Weave Net Docker Plugin](#setup-plugin)
+    * [Docker Compose File Syntax and the Weave Net Docker Plugin](#compose-plugin)
+
+##<a name="what-use"></a>What You Will Use
+
+* [Weave](http://weave.works)
+* [Weave Scope](http://weave.works/scope/index.html)
+* [Docker](http://docker.com)
+* [Docker Compose](https://www.docker.com/docker-compose)
+
+##<a name="before"></a>Before You Begin
+
+If you are using Windows, you can install [Docker for Windows](https://docs.docker.com/engine/installation/windows/), and if you are on a Mac, install [Docker for Mac](https://docs.docker.com/engine/installation/mac/) which provides all of the tools you need to complete this guide.
+
+For other operating systems, please install and configure the following separately before proceeding:
+
+  - [`docker-machine`](http://docs.docker.com/machine/#installation) binary (_`>= 0.8.0`_)
+  - [`docker`](https://docs.docker.com/installation/#installation) binary, at least the client (_`>= v1.6.x`_)
+  - [VirtualBox](https://www.virtualbox.org/wiki/Downloads) (_`>= 4.3.x`_)
+  - `curl` (_any version_)
+
+##<a name="attaching-containers"></a>Docker Compose and Attaching Containers to a Weave Network 
+ 
+ There are two different ways that you can attach containers to a Weave network when using Compose (which method to use is 
+ entirely up to you):
+ 
+ **1.** The Weave Net Docker API Proxy. See [Integrating Docker via the API Proxy](https://www.weave.works/docs/net/latest/weave-docker-api/).  
+ 
+ **2.**  The Docker Network Plugin framework. The Docker Network Plugin is used when 
+ Docker containers are started with the --net flag, for example: 
```

Comment:
For consistency with point 1, link to the plugin docs here too. (https://www.weave.works/docs/net/latest/features/#plugin)